paystack/build.gradle

```diff
@@ -31,6 +71,11 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$versions.kotlin"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$versions.coroutines"
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-android:$versions.coroutines"
+
+    testImplementation 'junit:junit:4.13.2'
```

Comment:
Should we use version as how we have above ?

Reply:
So @michael-paystack  is fixing this in hes PR over [here](https://github.com/PaystackHQ/paystack-android/pull/127) and he will fix the conflicts because hes a nice human 😂